tests/object/map.js

```diff
@@ -0,0 +1,15 @@
+var chai = require('chai');
+var expect = chai.expect;
+require('../../object/map');
+
+describe('map', function() {
+
+  it('should loop through results', function() {
+    var object = {name: 'zach', full: 'zach silveira'}
+    object.map(value => {
+      expect(value).to.match(/zach/)
+    })
```

Comment:
This tests that the value is getting passed into the function, but not really that it works as expected.